articles.tex

```diff
@@ -116,11 +116,11 @@
 \asubsection{Maintainer Expectations}
 Maintainers are expected to:
 \begin{itemize}
-    \item Review changes to the Constitution for grammar, spelling, and internal consistency.
-    \item Keep a public record of changes to the Constitution.
-    \item Participate in discussion of proposals.
-    \item Facilitate the creation of new changes to the Constitution by other members.
-    \item Be knowledgeable about the Constitution.
+	\item Review changes to the Constitution for grammar, spelling, and internal consistency.
```

Comment:
Is this indentation style consistent with the rest of the document or only the lines you touched?